### DISCUSSION TITLE
Project branding

### DISCUSSION THREAD

**Discussion by MarcinOrlowski** [logisim-evolution/logisim-evolution#1052] [General]
This thread is to discuss how visually project should be represented, incl. logo, icons, colors, fonts etc.
For reference, our current logo an icon is this:

![Logisim-evolution](https://raw.githubusercontent.com/logisim-evolution/logisim-evolution/master/artwork/logisim-evolution-logo.svg)

Before 3.6.0 we were using old icon original Logisim used (same as other forks used).

**Comment by R3dst0ne** [comment]
Some points where made in #884.
